v3/src/components/graph/models/graph-content-model.ts

```diff
@@ -139,6 +169,35 @@ export const GraphContentModel = DataDisplayContentModel
         xScale,
         yScale
       }
+    },
+    binDetails(options?: { shouldComputeInitial?: boolean }) {
```

Comment:
```suggestion
    binDetails(options?: { initialize?: boolean }) {
```
Seems just as clear and more concise.